misc/generate-flow.js

```diff
@@ -0,0 +1,68 @@
+const fs = require('fs-extra')
+const path = require('path')
+const util = require('util')
+const _ = require('lodash')
+const jsdoc = require('jsdoc-api')
+
+const ensureDir = util.promisify(fs.ensureDir)
+const remove = util.promisify(fs.remove)
+const writeFile = util.promisify(fs.writeFile)
+
+const generateFlow = async () => {
+  try {
+    const rootDir = path.resolve(__dirname, '..')
+    const generatedDir = path.resolve(rootDir, 'generated')
+    const flowDir = path.resolve(generatedDir, 'flow')
+    await remove(generatedDir)
+    await ensureDir(flowDir)
+
+    const items = await jsdoc.explain({
+      configure: path.resolve(rootDir, 'jsdoc.json'),
+      files: path.resolve(rootDir, 'src'), // Workaround while this hasn't been merged : https://github.com/jsdoc2md/jsdoc-api/pull/9
+    })
+
+    const itemsByNamespace = _.chain(items)
+      .filter('flow')
+      .groupBy('memberof')
+      .value()
+
+    const namespaces = _.keys(itemsByNamespace)
+    await Promise.all(namespaces.map(async namespace => {
+      const nsDir = path.resolve(flowDir, namespace)
+      await ensureDir(nsDir)
+
+      const nsItems = itemsByNamespace[namespace]
+
+      nsItems.forEach(async ({ name }) => await writeFile(
+        path.resolve(nsDir, `${name}.js`),
+        `import { ${name} } from '${namespace}/${name}'
```

Comment:
indentation is a bit weird here 😕 and in further generated code parts.

Reply:
I know but it allows to have a correct indentation in the generated files.
I haven't found a simple way of having a nicer indentation of the templates...
I think we can let it as it is and try to improve it later ?